src/main/resources/application-quickstarters.properties

```diff
@@ -0,0 +1,55 @@
+# Backend Quickstarter
```

Comment:
The name of this file seems to differ from what you load above?

Reply:
yes, this is on purpose, `src/main/resources/application-quickstarters.properties` defines a spring profile. It can can be optionally enabled i.e. when running the application locally.